spec/gitlab/client/projects_spec.rb

```diff
@@ -66,8 +142,6 @@
     end
 
     it 'returns information about a created project' do
-      expect(@project.name).to eq('Gitlab')
-      expect(@project.owner.name).to eq('John Smith')
```

Comment:
Why are these deleted?

Reply:
Don't knbow, it was a mistake. I add it back.